audinterface/core/process.py

```diff
@@ -171,21 +171,9 @@ def process_func(signal, _):
         if win_dur is not None and hop_dur is None:
             hop_dur = utils.to_timedelta(win_dur, sampling_rate) / 2
 
-        # figure out if special arguments
-        # to pass to the processing function
         signature = inspect.signature(process_func)
-        process_func_args = process_func_args or {}
-        self._process_func_special_args = {
-            'idx': False,
-            'root': False,
-            'file': False,
-        }
-        for key in self._process_func_special_args:
-            if (
-                    key in signature.parameters
-                    and key not in process_func_args
-            ):
-                self._process_func_special_args[key] = True
+        self._process_func_signature = signature.parameters
+        r"""Arguments present in processing function."""
```

Comment:
@frankenjoe 

If you are curious, I think this might be the only change in `audinterface` 1.2.0, that touches attributes of the objects.